### ISSUE TITLE
dialog: contains invisible elements that the screen reader announces

### ISSUE THREAD

**Issue by arubinoff** [angular/material#10570]
**Actual Behavior**:
 -  `What is the issue?` 
Inside the md-dialog-container are two divs with class `md-dialog-focus-trap` that are invisible on the screen, but detectable by a screen reader. Using ChromeVox on Chrome, and using the chromevox meta key + arrow keys, I was able to select these elements and hear "button" announced.  
Example of hidden element being selected:  
![image](https://cloud.githubusercontent.com/assets/3931713/24727349/a8e5a74e-1a23-11e7-87b9-4dc97a4b63bd.png)

 -  `What is the expected behavior?` These elements should be hidden from the screenreader.

**Steps to reproduce the issue**:
 -  This is reproducible in the demo page (https://material.angularjs.org/latest/demo/dialog) with the ChromeVox extension enabled.
- Steps:
  1. Open the dialog
  2. Select the dialog description text.
  3. Hit ChromeVox + Right twice
  4. Navigating around with the arrow keys will similarly select the left hidden element.

**AngularJS Versions**: 
 -  `AngularJS Version:` 1.1.3
 -  `AngularJS Material Version:` 1.1.3

**Additional Information**:
 -  `Browser Type: ` Chrome
 -  `Browser Version: ` 57
 -  `OS: ` Ubuntu

**Comment by EladBezalel** [comment]
@arubinoff can you tell which element is it?
+ it would be really nice if you'd do a pr to fix this issue!

**Comment by Splaktar** [comment]
Just as a note, I am not able to reproduce this in VoiceOver on macOS.

**Comment by Splaktar** [comment]
I am not able to reproduce this using Chromevox on ChromeOS Version 68.0.3440.87 (Official Build) beta (64-bit).

**Comment by Splaktar** [comment]
Based on our testing and [discussion](https://github.com/angular/material/pull/11400), it appears that this was fixed in Chromevox and is no longer reproducible.

If you are still seeing this issue, please let us know and provide updated reproduction steps and OS/browser/version info.
